src/main/scala/org/broadinstitute/dsde/firecloud/model/Workspace.scala

```diff
@@ -34,6 +34,14 @@ case class PfbImportRequest(
   workspace: Option[WorkspaceName] = None,
   user: Option[WorkbenchUserInfo] = None)
 
+case class ImportServiceRequest(
+  path: String,
+  filetype: String)
+
+case class ImportServiceResponse(
+  id: String,
+  status: String)
```

Comment:
I think it's probably an oversight in import service if we're not also returning an error message if one exists.